packages/aws-cdk-lib/aws-rds/lib/aurora-cluster-instance.ts

```diff
@@ -450,11 +450,13 @@ class AuroraClusterInstance extends Resource implements IAuroraClusterInstance {
 
     const isOwnedResource = Resource.isOwnedResource(props.cluster);
     let internetConnected;
-    let publiclyAccessible = props.publiclyAccessible;
+    let publiclyAccessible;
```

Comment:
```suggestion
    let publiclyAccessible = props.publiclyAccessible;
```
Is the `else` statement necessary?